clusterloader/glide.yaml

```diff
@@ -0,0 +1,23 @@
+package: github.com/sjug/perf-tests/clusterloader
```

Comment:
Awesome that you included glide, make sure to add README w/ instructions on how to build ! or maybe a dockerfile also.